src/test/java/uk/gov/hmcts/reform/sscs/ccd/domain/BenefitTest.java

```diff
@@ -15,4 +19,43 @@ public void givenAValidBenefitString_thenReturnTrue() {
     public void givenAnInvalidBenefitString_thenReturnTrue() {
         assertFalse(Benefit.isBenefitTypeValid("BLA"));
     }
+
+    @Test
+    @Parameters({
+            "PIP, Personal Independence Payment (PIP)",
+            "ESA, Employment and Support Allowance (ESA)",
+            "UC, Universal Credit (UC)",
+            "JSA, Job Seekers Allowance (JSA)",
+            "DLA, Disability Living Allowance (DLA)"
+    })
+    public void givenABenefitCodeWithAcronym_thenBuildLongBenefitNameDescriptionWithAcronym(String benefitCode, String expected) {
```

Comment:
Does this exercise the scenario where there is no Acronym? e.g. Carers Allowance?

Reply:
That's covered in the test below (givenABenefitCodeWithNoAcronym_thenBuildLongBenefitNameDescriptionWithNoAcronym) although thinking about it, maybe i don't need to split into separate tests. What you think?